static/src/javascripts/projects/common/modules/discussion/discussion-frontend.js

```diff
@@ -49,6 +56,20 @@ define([
         });
     }
 
+    function otherBannersVisible () {
+        var siteMessage = document.getElementsByClassName('js-site-message');
+        if (siteMessage.length && !siteMessage[0].classList.contains('is-hidden')) {
+            // Contribution banner is visible
+            return true;
+        }
+        var breakingNews = document.getElementsByClassName('js-breaking-news-placeholder');
+        if (breakingNews.length && !breakingNews[0].classList.contains('breaking-news--hidden')) {
+            // Breaking news alert is visible
+            return true;
+        }
+        return false;
+    }
+
```

Comment:
There are a couple of mediator events (one on the membership banner, one on the adblock banner) that may be worth waiting for, so that this banner doesn't beat them to the slot (they both have async requests).

How does it behave with the Weekend Reading banner (#14218) and the US survey banner (#14255)? Does it beat them to the slot?